src/pages/Projects/styles/Projects.styled.ts

```diff
@@ -26,6 +26,16 @@ export const Image = styled.img`
   height: 20%;
   width: 15%;
 `
+export const DividerContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 80px;
+`
+
+export const Divider = styled.div`
```

Comment:
since it is a line, would the hr element be better?

Reply:
resolved in [df507ae](https://github.com/rhdevs/rhdevs-website/pull/74/commits/df507ae998faf1d5fa158800005deb812bd57ee4)